tox.ini

```diff
@@ -5,13 +5,14 @@ envlist = py{34,35,36}-test, checkstyle, docs
 usedevelop = True
 deps =
      test: pytest
+     test: -rrequirements.txt
```

Comment:
Why did you add this part? 
For me everything works (locally), even if i comment it out. I think, tox uses the package install (setup.py) to install the dependencies via requirements.txt anyway

Reply:
I added this line because it did not work for me without. @derNarr didn't you have the same problem as well?

However, I am currently not able to reproduce the error anymore